### ISSUE TITLE
'saml2aws exec' breakages since 2.0.3

### ISSUE THREAD

**Issue by CaseyLeask** [Versent/saml2aws#127]
I've tried multiple versions across ubuntu and os x and we're all receiving the same error since 2.0.3

saml2aws configure
Please choose the provider you would like to use:
[ 0 ]:  ADFS
Please choose an MFA you would like to use:
[ 0 ]:  Auto

```sh
$ env | grep profile
AWS_PROFILE=saml
```

Oddly, it works fine for `saml2aws login`

```sh
$ saml2aws login
$ cat ~/.aws/credentials
[saml]
aws_access_key_id     = <valid>
aws_secret_access_key = <valid>
aws_session_token     = <valid>
aws_security_token    = <valid>
```

```sh
$ tar -xvf saml2aws_2.1.0_darwin_amd64.tar.gz
x saml2aws
$ mv saml2aws $(which saml2aws )
$ saml2aws exec -- aws s3 ls
Running command as: arn:aws:sts::xxx:assumed-role/xxx/caseyl

An error occurred (AuthorizationHeaderMalformed) when calling the ListBuckets operation: The authorization header is malformed; a non-empty Access Key (AKID) must be provided in the credential.
exit status 255

$ tar -xvf saml2aws_2.0.2_darwin_amd64.tar.gz
x saml2aws
$ mv saml2aws $(which saml2aws )
$ saml2aws exec -- aws s3 ls
Running command as: arn:aws:sts::xxx:assumed-role/xxx/caseyl
<healthy results>
```

**Comment by chamila-c** [comment]
@CaseyLeask  yeah, sorry, pretty sure this one is my bad, having broken `exec` with my changes in https://github.com/Versent/saml2aws/pull/108

But hopefully should be fixed with https://github.com/Versent/saml2aws/pull/125 (thanks to @dan-v !)

Might just need @wolfeidau to push out a new release that includes the fix 🙏

**Comment by wolfeidau** [comment]
Just published a new release 2.2.0 with a few fixes and RPM / DEB packages.

https://github.com/Versent/saml2aws/releases/tag/v2.2.0

Any issues please update this ticket.
